Slightly expand Home page (#86)
kasenvr/vircadia-docs-sphinx: docs/source/home.rst

Home
####

> Vircadia is an open-source software which allows you to create and share virtual reality (VR) experiences. You can create and host your own virtual world,
> explore other worlds, meet and connect with other users, attend or host live VR events and much more.

Reviewer: ```suggestion
Vircadia is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
explore other worlds, meet and connect with other users, attend or host live VR events, and much more.
```

The Vircadia metaverse provides built-in social features, including avatar interactions, spatialized audio and interactive physics.
Additionally, you have the ability to import any 3D object into your virtual environment. No matter where you go in Vircadia,
you will always be able to interact with your environment, engage with your friends, and listen to conversations just like you would in real life.

> :doc:`Get Vircadia <explore/get-started/install>` |Download| or take a look at our :doc:`Gallery <explore/gallery>` |Gallery|

Reviewer: ```suggestion
:doc:`Get Vircadia <explore/get-started/install>` |Download| or take a look at our :doc:`Gallery <explore/gallery>` |Gallery|.
```

.. |Download| image:: _images/remixicon/download-2-fill.svg
              :class: inline
              :width: 20

.. |Gallery| image:: _images/remixicon/gallery-fill.svg
              :class: inline
              :width: 20

> In addition to the documentation, there is also a `Discord server <https://discordapp.com/invite/Pvx2vke>`__ |Discord| available.

Reviewer: ```suggestion
In addition to the documentation, you can join the discussion on the Vircadia `Discord server <https://discordapp.com/invite/Pvx2vke>`__ |Discord|.
```

Home
####

Vircadia is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
explore other worlds, meet and connect with other users, attend or host live VR events, and much more.

> The Vircadia metaverse provides built-in social features, including avatar interactions, spatialized audio and interactive physics.

Reviewer 2: ```suggestion
The Vircadia metaverse provides built-in social features, including avatar interactions, spatialized audio, and interactive physics.
```

Author: My teachers would have given me shit for adding an *optional* comma
haha

Reviewer 2: Heh, the reason for that is consistency. The rule is you don't have to do it, but if you do in one place then you've gotta do it everywhere ideally. :) I personally do it as well, to me it makes the English more clear, so that makes two of us.
